File: 1-iniciante/1043/1043.lua

a, b, c = io.read("*n","*n","*n")

if math.abs(b-c) < a and a < b + c
and math.abs(a-c) < b and b < a + c
and math.abs(a-b) < c and c < a + b then
	print (string.format("Perimetro = %0.1f", a + b + c))
else
	print (string.format("Area = %.1f", (a + b) * c / 2))
end

File: 1-iniciante/1043/1043.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    double a, b, c;
    scanf("%lf%lf%lf", &a, &b, &c);

    if (abs(b - c) < a && a < b + c &&
    abs(a - c) < b && b < a + c &&
    abs(a - b) < c && c < a + b)
        printf("Perimetro = %.1lf\n", a + b + c);
    else
        printf("Area = %.1lf\n", (a + b) * c / 2);
    return 0;
}
